### DISCUSSION TITLE
开启软件后,时不时假死

### DISCUSSION THREAD

**Discussion by msccreater** [shuax/MouseInc.Settings#449] [Q&A]
开启软件后,在移动鼠标时,就会出现假死现象,找不到鼠标位置,来回移动鼠标位置大概5秒左右才会出现鼠标图标,并可以正常移动鼠标,关闭软件后,鼠标假死现象消失....
好像用,但是经常假死,有解决办法吗?

软件版本: 2.13.4 A

版本	Windows 11 专业版
版本	21H2
安装日期	‎2022/‎4/‎19
操作系统版本	22000.978
体验	Windows 功能体验包 1000.22000.978.0

**Comment by shuax** [comment]
可能不兼容吧，建议换别的
